### PR TITLE
Clarify use of template shorthand with if-then

### PR DESCRIPTION
**Pull request by TomBrien**
## Proposed change

When using if-then(-else) block, you can't use a template shorthand condition as the value to the `condition` key. Instead it has to be used directly with the `if`. Currently the template shorthand documentation can lead to confusion as we imply (out right say) you can use shorthand wherever there is a `condition(s)` key, This adds an example to the if-then docs to clarify the correct usage.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/scripts.markdown [hunk 1]
```diff
@@ -604,6 +604,22 @@ This action supports nesting, however, if you find yourself using nested if-then
 actions in the `else` part, you may want to consider using
 [choose](#choose-a-group-of-actions) instead.
 
+[Template shorthand notation](https://www.home-assistant.io/docs/scripts/conditions/#template-condition-shorthand-notation) can be used with if-then. The code below is equivalent to the example above.
```
Comment (frenck) [inline source/_docs/scripts.markdown]: Don't use full links

File: source/_docs/scripts.markdown [hunk 2]
```diff
@@ -604,6 +604,22 @@ This action supports nesting, however, if you find yourself using nested if-then
...
+
+```yaml
+script:
+  - if: "{{ is_state('zone.home', 0) }}"
```
Comment (frenck) [inline source/_docs/scripts.markdown:613]: This feels misplaced? As in, like an oddity to mention in the middle out of the blue?
This page is not aimed at showing how conditions work.

The conditions page has IMHO a perfectly fine explanation for this already?

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
> When using if-then(-else) block, you can't use a template shorthand condition as the value to the condition key

I think you are confusing a "condition" versus a list of c `conditions` (the array).

That sounds weird and not correct either? 


> Currently the template shorthand documentation can lead to confusion as we imply (out right say) you can use shorthand wherever there is a condition(s) key,

That is not this page, so not sure why it is "fixed" on this page either.

Additionally, I don't see the confusion?

../Frenck

**Comment by TomBrien** [comment]
Currently we say "This shorthand notation can be used everywhere in Home Assistant where conditions are accepted" and have examples on the conditions docs showing short hand in place of either the whole list `conditions: "{{ ... }}"` and also in place of a single condition.

```yaml

conditions:
  - "{{ (state_attr('device_tracker.iphone', 'battery_level')|int) > 50 }}"
  - condition: state
    entity_id: alarm_control_panel.home
    state: armed_away
  - "{{ is_state('device_tracker.iphone', 'away') }}"
```

There is also a specific example that talks about condition actions (I agree not the same thing as here). That shows `- condition: {{ ... }}`.

My reasoning for clarifying here was not to dilute the conditions page with a specific example for this syntax since, to me at least, it's not clear that `if` is equivalent to `conditions`. I assumed people struggling with and if-then would come to this page. I'm happy to make changes to that page or just to close this. Maybe it's just my confusion but I'm working on the concept that if I got confused others probably have too.

**Comment by frenck** [comment]
> This shorthand notation can be used everywhere in Home Assistant where conditions are accepted

That is correct 🤷 

> There is also a specific example that talks about condition actions (I agree not the same thing as here). That shows - condition: {{ ... }}.

There is no such thing as a condition action.

>  it's not clear that if is equivalent to conditions. 

I don't agree with that. The paragraph explains that in the first line? "This action allows you to conditionally (`if`)..."

> I assumed people struggling with and if-then would come to this page.

I've not came across such a case at this point before, to be honest.

../Frenck

File: source/_docs/scripts.markdown (hunk 1 above)
Comment (TomBrien) [reply source/_docs/scripts.markdown]:
```suggestion
[Template shorthand notation][shorthand-template] can be used with if-then. The code below is equivalent to the example above.
```

File: source/_docs/scripts.markdown (hunk 2 above)
Comment (TomBrien) [reply source/_docs/scripts.markdown:613]: We have something pretty much equivalent for [Repeat-until](https://www.home-assistant.io/docs/scripts/#repeat-until) on this page, which is why I thought here was sensible

File: source/_docs/scripts.markdown [hunk 3]
```diff
@@ -604,6 +604,22 @@ This action supports nesting, however, if you find yourself using nested if-then
...
+    then:
+      - alias: "Then start cleaning already!"
+        service: vacuum.start
+        target:
+          area_id: living_room
+    else:
+      - service: notify.notify
+        data:
+          message: "Skipped cleaning, someone is home!"
+```
```
Comment (TomBrien) [inline source/_docs/scripts.markdown:621]:
```suggestion
{% raw %}

```yaml
script:
  - if: "{{ is_state('zone.home', 0) }}"
    then:
      - alias: "Then start cleaning already!"
        service: vacuum.start
        target:
          area_id: living_room
    else:
      - service: notify.notify
        data:
          message: "Skipped cleaning, someone is home!"
```

{% endraw %}
```

**Comment by TomBrien** [comment]
Fair enough

**Pull request closed without merging**